src/app/tide/peptide_mods3.cc

```diff
@@ -61,275 +61,251 @@ class IModsOutputter {
   virtual bool GetTempFileNames(vector<string>& filenames) = 0;
 };
 
-// Alternative class to generate modified peptides. Writes to temporary files
-// before merging them. Bins modified peptides by mass, then writes them to the
-// temporary file associated with that bin; the number of temporary files
-// required is therefore bounded by the variance in peptide masses, rather than
-// the number of possible modifications.
-class ModsOutputterAlt : public IModsOutputter {
+// Original class to generate modified peptides. Writes to temporary files
+// before merging them. As the number of possible modifications increases, the
+// number of required temporary files can grow extremely large.
+class ModsOutputter : public IModsOutputter {
```

Comment:
If we are getting rid of the alternate class. Does it make sense to change the class name to the original version? Before Kaipo edited it this line looked like the below.

"class ModsOutputter {"

Reply:
Removed.